### ISSUE TITLE
Gantt chart view - choose timezone

### ISSUE THREAD

**Issue by ColemanTom** [cylc/cylc-ui#1917]
### Problem

I was looking at the Gantt view to see timings and was getting confused because the time along the x-axis was local time. I was expecting UTC because that is what I was wanting - workflow is triggered/runs using UTC, so analysis of it I tend to do in UTC.

### Proposed Solution

Provide the ability to choose timezone, or at least a switch to allow changing between Local and UTC. Or perhaps, have both visible along the x-axis.

Also, the x-axis should not just say `Time`, it should be `Time (tz)` where tz is the display timezone.

**Comment by ChrisPaulBennett** [comment]
Ah, yes. Slight oversight on my part. I'm currently swamped at the moment. If someone else wants to deal this, you are very welcome to. Other wise I'll try and fit this in

**Comment by ChrisPaulBennett** [comment]
I've had a look at this and none of my code is altering the time. I get the time values from a GraphQL query, which gives me the time in Zulu time (I'm assuming that's what the z means). Zulu time is the same as UTC.
The x-axis is auto-generated by chart library (I will guess the scale from the max and min times in the data). So the x-axis will always match the data.

Can you try the same GraphQL commands as shown below.

Are you sure that you are getting local time?
If you definitely are, I think it might be graphQL giving you time based on your time zone

![image](https://github.com/user-attachments/assets/4a1297b2-f139-4a08-8979-55cc3eba633e)

**Comment by ColemanTom** [comment]
> Are you sure that you are getting local time?

Definitely. I just ran a suite, Gantt chart is sowint 905am instead of 2305UTC

>  If you definitely are, I think it might be graphQL giving you time based on your time zone

When I run your query, it returns the results in UTC time, not local time. 

![image](https://github.com/user-attachments/assets/c132b0c7-9472-428d-857e-08e8bb1ec2fe)

JS frontend perhaps?

**Comment by ChrisPaulBennett** [comment]
Curious. I'll have another look

**Comment by ChrisPaulBennett** [comment]
The JavaScript functions that get time all convert to local time by default (Which should have been obvious to me).
x-axis strings are now UTC 
https://github.com/cylc/cylc-ui/pull/1946

**Comment by MetRonnie** [comment]
To summarise what is left to do:
- (cylc-flow) expose UTC mode via the GraphQL schema so it can automatically be chosen by the Gantt view
- Dropdown to choose between local time or UTC?
